### PR TITLE
Hack Gitian descriptors and keys into vaguely the right shape

### PR DESCRIPTION
**Pull request by rnicoll**
Note I've removed Leofidus' key due to inactivity, and added a weight of 20 to whit's key.

### CODE REVIEW

File: contrib/gitian-downloader/win32-download-config [hunk 1]
```diff
@@ -1,42 +1,29 @@
...
+  90DC2AC256201AD9377E15049142E5F7E533CE3B:
+    weight: 40
+    name: rnicoll
+    key: rnicoll
+  1DDC450B45DB5ADCCF5DDA7F8E4217C6D47D946D:
+    weight: 40
+    name: langerhans
+    key: langerhans
+  6E9C3630F63C1541054C9503B1C60A65E5E64D7B:
+    weight: 20
```
Comment (langerhans) [inline contrib/gitian-downloader/win32-download-config]: Guess that should be 40 too? (Same in linux file)

Comment (rnicoll) [inline contrib/gitian-downloader/win32-download-config]: Think @patricklodder specifically wanted less weight. It's consistent with the previous value, anyway.

Comment (langerhans) [inline contrib/gitian-downloader/win32-download-config]: Hm, now that you say it, I remember something along those lines. Also, the downloader is not in use anyway.

**Comment by patricklodder** [comment]
Missing stapler's key?

**Comment by rnicoll** [comment]
Err... I can see his key and weight in both of the downloader configs, am I missing something?

**Comment by patricklodder** [comment]
you're right. need more :coffee: i guess :(

**Comment by langerhans** [comment]
utACK

**Pull request merged**